src/group_theory/submonoid/membership.lean

```diff
@@ -2,7 +2,7 @@
 Copyright (c) 2018 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Kenny Lau, Johan Commelin, Mario Carneiro, Kevin Buzzard,
-Amelia Livingston, Yury Kudryashov
+Amelia Livingston, Yury Kudryashov, Alex Zhao
```

Comment:
My impression is that you shouldn't add your name to the authors list if you are adding a single lemma to a large file.

Reply:
Ok, I've changed it back.